docs/source/topics/con_about-presets.adoc

```diff
@@ -0,0 +1,18 @@
+[id="about-presets_{context}"]
+= About presets
+
+[role="_abstract"]
+{prod} presets represent a managed container runtime and the lower bounds of system resources required by the instance to run it.
+{prod} offers presets for {ocp} and the Podman container runtime.
```

Comment:
no variable for Podman?

Reply:
I didn't see the need to create one. The attributes for OpenShift largely exist to avoid unfortunate typos when the "f" key is missed. ;)

If we foresee the name of Podman itself (or the `podman` preset) to change, we should probably create an attribute for it.